scripts/acceptance.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+readonly PROGDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly DEPSERVERDIR="$(cd "${PROGDIR}/.." && pwd)"
+
+go test -short -v "${DEPSERVERDIR}"/pkg/acceptance
```

Comment:
I think this should be `"${DEPSERVERDIR}"/pkg/dependency/acceptance`

Also I think we want to remove the `-short` flag from the acceptance script, otherwise it doesn't do anything.

Reply:
You are correct. This was a bad copy/paste. 

I can't actually run the acceptance tests since I haven't found a `GITHUB_TOKEN` with a minimal set of permissions that enables the test to run. Also, running with a `GITHUB_TOKEN` without the appropriate permisisons takes close to 10 minutes to timeout. Once I find the appropriate permissions I'll PR them into some kind of repo docs.